### ISSUE TITLE
Write copy for Projects Page Header

### ISSUE THREAD

**Issue by daniellex0** [hackforla/website#1051]
### Overview
We need to design a header for the new projects page (which will be linked to from the homepage). The body of the page will be very similar to the current homepage.

### Action Items

**UX/UI**
- [x] Replicate the common header layout (with side-by-side heading and vector animation) shown in the [design system ](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)(bottom row to the left)
   - [x] Follow the precise margins and sizes to keep consistency on all pages
   - [x] see [Join us ](https://www.hackforla.org/join)and [Sitemap](https://www.hackforla.org/sitemap/) pages etc for live examples of this header layout
- [x] Design new desktop projects page header on [Figma file titled Project Page (WIP)](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=532%3A1454)
- [x] Design new mobile projects page header
- [x] Swap hero illustration
- [x] Run mockup by design team

**UX Writing**
- [ ] Write description of Projects Page

**Development**
- [x] Hand off to development

### Resources/Instructions
[Project Page (WIP) Figma File](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=532%3A1454)
[Freepik Stories](https://www.freepik.com/stories) 

**Guidance for UX Writing**

***What does this page need to say in order for it to be useful?***
- We need this page to introduce our projects in such a way that it encourages the manner of engagement we want from potential volunteers. 
- The text at the top doesn’t reflect the path we want them to follow, we don’t want them to pick a project and start from there. We want them to follow the Join Us link. Letting people go directly to projects is hurting our projects. 
-  A current issue, for example, is that a lot of devs have recently joined a certain project. The problem is they’re all very junior and en masse are not able to support the project (and support their own growth) as well as if there were a mix of different experience level devs on the project. This is why we don’t want people to come to the site and get married to the idea of a certain project, we want them to get excited about what we do on a larger scale, keep an open mind, then go through our intended onboarding flow and then from there get matched up to an appropriate project. 

***The use of this page we do want to encourage:***
- Finding out about our cool projects and getting excited about what we do
- Being wowed at how many there are / their breadth

***The use we don’t want to encourage:***
- Using this page to find a project that potential volunteers want to work on specifically. We instead want them to go to an onboarding session, join their CoP, and through this flow to get matched up with a project they would be good for. 
- Everyone who wants to join a project MUST first join onboarding. We want them to start there (onboarding), not just get their mind set on a project through this page and then try to join that.

**Comment by shinhope** [comment]
Progress: n/a
Blocks: midterm exams, busy week
Availability: next Tues-Thu
ETA: by next Friday

**Comment by daniellex0** [comment]
Assigning to @qtsash because this is time sensitive (and @shinhope is assigned another issue)

**Comment by qiqicodes** [comment]
@SaashaG Can you let us know your progress in regards to this issue? Thank you =]

Progress:
Blockers:

**Comment by SaashaG** [comment]
I will work on it today and let you know. Thanks

**Comment by SaashaG** [comment]
I have started the header but would love to get some feedback. @daniellex0 let me know when you have time to discuss.

**Comment by SaashaG** [comment]
Progress: 50%
Availability: Sunday 5/16
ETA: 5/15

![image](https://user-images.githubusercontent.com/83323065/118170453-19bcd880-b3f8-11eb-8455-6cad71df2e43.png)

**Comment by daniellex0** [comment]
This is looking really good, thank you for following the design system! I know there are a ton of tiny details to follow that are easy to miss, so I made a few adjustments (made the paragraph line height 137%, and adjusted some spacing). 

Can you make the following adjustments:

- [x] In mobile, change "Join Us" to the new page title
- [x] In desktop, maybe the paragraph width 508px

Thank you!

**Comment by SaashaG** [comment]
I have updated the design...
![image](https://user-images.githubusercontent.com/83323065/118897310-fbfbe180-b8d7-11eb-8246-c20a00b7d873.png)

**Comment by daniellex0** [comment]
@SaashaG Perfect, looks good!

**Comment by Sihemgourou** [comment]
@daniellex0 this seems finished right?

**Comment by daniellex0** [comment]
@Sihemgourou Yes the design is finished and being developed- we just need to fix the copy (the paragraph in the header)

**Comment by ahoang94** [comment]
@kristine-eudey I can have this done by next Thursday's meeting, but I would first like some guidance as to the expectations for this new copy. What are we aiming for, compared to what's in the existing copy?

**Comment by kristine-eudey** [comment]
@ahoang94 I will update this issue by EOD on 8/15/21 with further clarification about expectations for the new copy.

**Comment by kristine-eudey** [comment]
@ahoang94 please see updated guidance for copy in the Resources/Instructions section of the issue, and let me know if you'd like to discuss or need more context.

**Comment by ahoang94** [comment]
1. Progress: No progress thus far, but this will change soon.
2. Blockers: There was some uncertainty about the aims of the copy, but I believe I now have it figured out.
3. Availability: Can work on this issue over the weekend.
4. ETA: I hope to have a first draft done by Sunday's meeting (8/22) at the earliest, or mid week at the latest.

**Comment by ahoang94** [comment]
@kristine-eudey I have contributed a few first draft options in the green box to the left of the most recent mockup (blue rectangle).

**Comment by kristine-eudey** [comment]
@ahoang94 Great, thanks! I'll take a look and share thoughts within the next day on the drafts you came up with. Moving this back to "In Progress" until it's completely finished.

**Comment by kristine-eudey** [comment]
Hi @ahoang94, these copy options are great and definitely get at what we need to communicate here. I placed your wording options within the page design and brought them to discuss with Bonnie and the PM team yesterday. A few tweaks/combinations were made, which you can see reflected in [versions 4a and 4b](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=14587%3A94671), with one including link in text and the other using a button for the CTA. Please take a look and let me know what you think of this iteration!

**Comment by ahoang94** [comment]
@kristine-eudey The copy has been revised in the Artboard "[4a v2](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=14743%3A95905)" (see orange box). Please note my comment to the left of the board and let me know if we should act accordingly.

**Comment by kristine-eudey** [comment]
Updated copy was shared with Bonnie and PM team on 8/31. Additional changes were identified as being needed. 

- PM team will identify and give UX team clarity on what the most important elements that need to be mentioned on the header of our Projects page. 
  - Question for PM team to answer: what do we want people to think about when they think about our projects? How do we want to frame our projects for them?

**Comment by ExperimentsInHonesty** [comment]
@ahoang94 This is the new copy for the page

Our projects have real impact in local communities.  Use the filters below to learn about our project and the teams by Status, Technology and Program area.

There are countless opportunities to build digital products, programs and services across a number of skill levels and practice areas. Visit the [Join Us](url) page to find out how you can get involved or see the [About Us](url) page to find out how we build all these project with volunteers.

**Comment by ahoang94** [comment]
I have updated the [Figma file](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15514%3A99250) to include a draft with Bonnie's updated copy.

**Comment by kristine-eudey** [comment]
@ahoang94 I worked on a version that incorporated the copy you wrote along with the changes requested by the product team, please see this newest version [here](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15634%3A99905). This copy was signed off on by Bonnie and PMs, but it needs work in terms of formatting/fitting the content in the header in a way that works. Please feel free to jump back in with this copy and see if you can get it to work visually. I added a grid to the page that matches that of the new homepage, please work to that grid where possible, while still not drifting too far from current design system requirements.

### Some discussion points I wanted to share based on the discussion in my meeting with Bonnie/pms:
- **I brought up that I thought the flow would be more natural if it was structured:**

"Hack for LA’s projects have real impact in local communities. Our projects also offer volunteers of all skill levels countless opportunities to build digital products, programs, and services across diverse fields of practice. 

Use the filters below to learn about our projects and the teams by Status, Technology, and Program area."

- **Feedback was offered that linking the first two sentences together might push the user behavior we're trying not to encourage: volunteers getting involved with the org via a project, rather than via general onboarding. So the order was switched to pull those two concepts apart from each other (exploring our projects / getting involved):**

"Hack for LA’s projects have real impact in local communities. Use the filters below to learn about our projects and the teams by Status, Technology, and Program area. 

Our projects also offer volunteers of all skill levels countless opportunities to build digital products, programs, and services across diverse fields of practice."

**Comment by ahoang94** [comment]
@kristine-eudey I have created [5 layout options](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15831%3A100334) for the updated copy!

**Comment by ExperimentsInHonesty** [comment]
@kristine-eudey I thought we either decided which one, or we were going to let the team vote.  But this didn't seem to make the agenda today.  Can you followup on this so that it has one version of the site in the red box.

**Comment by kristine-eudey** [comment]
When decision is made, add correct header to all new/current iterations in Figma.

**Comment by kristine-eudey** [comment]
Final version is complete and the Figma design can be found in the pink rectangle [here](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15831%3A100334)

@abuna1985 this is now ready for a dev issue.

**Comment by ahoang94** [comment]
@kristine-eudey @abuna1985 

I think there might be a discrepancy with the size of the artwork. It is currently set to 372px in height, but according to the [design system](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=6207%3A40530) it should be 240px. Kristine, please advise.

**Comment by kristine-eudey** [comment]
@ahoang94 thanks for spotting that! Can you correct it? I can then go in and copy the adjusted version over to all the other new designs that incorporate it.

**Comment by ahoang94** [comment]
@kristine-eudey The layout has been corrected in the "Final Version" artboard. I used Figma's auto layout feature to include built-in top and bottom margins (per the design system), which should help guide its placement on other artboards/designs:
![Screen Shot 2021-10-05 at 1 03 16 PM](https://user-images.githubusercontent.com/85914721/136094735-43f85c8d-f5db-45cf-8534-f3631a38767a.png)

I also updated the mobile version to have the approved copy + layout.

**Comment by kristine-eudey** [comment]
@ahoang94 looks great! See [final design here](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15831%3A100334). Closing issue as all design objectives have been met. This is now ready for development.

**Comment by adam-abundis** [comment]
@kristine-eudey Created #2355 to complete the dev portion. Moving this to the `Done` column
